config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java

```diff
@@ -137,6 +137,7 @@
     public static final String BINDINGS_OVERVIEW_HANDLER_CLASS = BindingsOverviewHandler.class.getName();
     public static final String STATE_HANDLER_CLASS = "com.yahoo.container.jdisc.state.StateHandler";
     public static final String STATISTICS_HANDLER_CLASS = "com.yahoo.container.config.StatisticsRequestHandler";
+    public static final String LOG_HANDLER_CLASS = "com.yahoo.container.handler.LogHandler";
```

Comment:
You can use the class object instead of a String for this class, because container-core is always available runtime and the package is exported.

Reply:
Nice, will change to do that

Reply:
Fixed